packages/studio-web/.github/workflows/codeql.yml

```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Do we still want to have the GH Actions workflows here?

Reply:
right, no. good catch.